src/components/LinkListItem/LinkListItem.tsx

```diff
@@ -73,7 +73,7 @@ export const LinkListItem = ({
       >
         {iconPosition === 'before' && (
           <Icon
-            aria-hidden="true"
+            purpose="decorative"
             name={iconName}
             className={styles['link-list__icon']}
```

Comment:
I bet some of these have colors set in the styles.. Maybe make a ticket to go back and clean those up?